src/core/server/integration_tests/ui_settings/routes.test.ts

```diff
@@ -47,7 +47,7 @@ describe('ui settings service', () => {
     describe('set', () => {
       it('validates value', async () => {
         const response = await request
-          .post(root, '/api/kibana/settings/custom')
+          .post(root, '/internal/kibana/settings/custom')
           .send({ value: 100 })
           .expect(400);
 
```

Comment:
We gonna need proper test coverage for the public routes too. Those tests needs to be running against the public routes (duplicating them is fine)

Reply:
Sure, I added a duplicate set of tests for both public and internal routes.